Lib/test/test_multiprocessing_main_handling.py

```diff
@@ -65,6 +65,13 @@ def f(x):
             raise RuntimeError("Timed out waiting for results")
     results.sort()
     print(start_method, "->", results)
+    sys.stdout.close()
```

Comment:
While I understand it's practical to do so, it's a bit weird to add your test there, as it doesn't really have anything to do with `__main__` handling. Instead, I would expect it to be somewhere in `_TestProcess` in `_test_multiprocessing.py`.

Reply:
I also wondered about the placement, so I think you should follow Antoine's suggestion.